app/templates/index.hbs

```diff
@@ -27,7 +27,7 @@
 
 <div class="emberconf-announcement call-to-action__box">
   <div class="logo-and-date">
-    <a href="//emberconf.com/">
+    <a href="//emberconf.com/" aria-label="EmberConf 2020 Logo">
```

Comment:
It should just be `aria-label="Ember conf 2020"`

Reply:
@MelSumner updated this!